### ISSUE TITLE
Missing definition/declaration file (.d.ts)

### ISSUE THREAD

**Issue by PyroGenesis** [s-yadav/iv-viewer#43]
Hello,

A definition file (`.d.ts`) for this library would be appreciated.

Currently, TypeScript cannot infer any types (defaults to `any` type) on import. And running `npm install @types/iv-viewer` results in a 404.

[For reference](https://github.com/microsoft/typescript/issues/3019)

Thanks.

**Comment by s-yadav** [comment]
Yes there are no types defined yet. Can you raise a PR for the definition file?

**Comment by PyroGenesis** [comment]
Sure, I'll give it a go.
